README.md

```diff
@@ -99,17 +99,45 @@ const tuple5 = tuple4.popped();
 assert(tuple5 === #[2, 2, 3, 4]);
 ```
 
-#### Computed access
+#### Record initialization
+
+Computed keys are supported, similar to [computed property names](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names) in object literals.
+
+```js
+const key = "a";
+assert(#{ [key]: 1 } === #{ a: 1 })
+assert(#{ [key.toUpperCase()]: 1 } === #{ A: 1 })
+```
+
+Non-string keys are coerced to strings.
+
+```js
+assert(#{ [true]: 1 } === #{ true: 1 })
+assert(#{ [true]: 1 } === #{ ["true"]: 1 })
+
+assert(#{ [1 + 1]: "two" } === #{ 2: "two" })
+assert(#{ [9 + 1]: "ten" } === #{ ["10"]: "ten" })
+
+assert(#{ [window.location]: 1 } === #{ [String(window.location)]: 1 })
```

Comment:
I want to confirm [`ToPropertyKey` ](https://github.com/tc39/proposal-record-tuple/issues/60#issuecomment-576369581) would follow this behaviour?

Reply:
I don't understand why you're referencing `window.location`--this seems like a bit of a distraction. Anyway, yes, it will convert it to a string.

Reply:
I thought it convenient way to show overridden `.toString()` being invoked. But, it's not very clear, now that I'm taking another look at it.

Would you say this is more effective?

------

Overridden `toString()` methods are invoked.

```
const obj = {
  value: 'a',
  toString() { return this.value }
}

assert(#{ [obj]: 1 } === #{ a: 1 })

obj.value = 'b'
assert(#{ [obj]: 1 } !== #{ a: 1 })
```